### PR TITLE
Campus API

### PR DESCRIPTION
**Pull request by jhawkzz**
Added IsOnCampus API
Added Campus API for retrieving campus models

### CODE REVIEW

**Comment by jhawkzz** [comment]
Same pull request as before--It was just easier to re-create it rather than merge with master after the cleanup I just did.

File: App.Shared/Network/API/Controller/ContentController.cs [hunk 1]
```diff
@@ -90,5 +90,100 @@ public static void PersonalizedContent( int numCampaigns, int? primaryAliasId, O
...
+        {
+            NotSet = -1,
+
+            Success,
+
+            CampusNotFound
+        }
+
+        public static void Campus( int? campusId, OnCampusResult onResult )
+        {
```
Comment (dwee9) [inline App.Shared/Network/API/Controller/ContentController.cs]: Why does this return a List of Campus as opposed to just a single campus?

Comment (jhawkzz) [reply App.Shared/Network/API/Controller/ContentController.cs]: I did that so the callback is consistent whether you pass in 'null' to get all campuses, or a value to get one campus.

If you pass null, you get a list of every campus. If you pass a campus Id, you'll get a list of one campus, or an empty list and an error if the campus doesn't exist.

I'm happy to change this to give you back just a single campus object when you pass a specific campusId if you'd like. Just let me know!

Comment (dwee9) [reply App.Shared/Network/API/Controller/ContentController.cs]: That works for me, I was just curious cause it looked like it was only getting one. But that makes sense

**Pull request merged**